src/jack.py

```diff
@@ -1440,6 +1440,24 @@ def shortname(self):
         """
         return _ffi.string(_lib.jack_port_short_name(self._ptr)).decode()
 
+    @property
+    def alias(self):
+        """Alias of the JACK port, not mandatory.
+
+        The alias string returned is something like that:
+        
+        in-hw-1-0-0-LPK25-MIDI-1
+        
+        or
+        
+        out-hw-2-0-0-MK-249C-USB-MIDI-keyboard-MIDI-
+
+        """
+        aliases=[_ffi.new("char[0xff]"), _ffi.new("char[0xff]")]
```

Comment:
You should probably use `port_name_size()` instead of `0xff`?

Reply:
I think port_name_size() return the size of the "name", not the alias. I've not found any way of getting the alias size, so i decided to use 255, that should be enough ... i hope ;-)

Reply:
Sorry, i just read the "set_alias" documentation and it says: